contrib/devtools/update-translations.py

```diff
@@ -159,6 +161,11 @@ def postprocess_translations(reduce_diff_hacks=False):
                 for translation in translations:
                     if translation is None:
                         continue
+
+                    # Check if btc address is included in translation after removing whitespaces
+                    if re.search(ADDRESS_RE, translation.replace(' ', '')):
+                        translation_node.set('type', 'malicious')
+                        break
```

Comment:
nit: I still like the idea of using errors for this.

Reply:
Yes..realized now. Amending.